README.md

```diff
@@ -15,51 +16,156 @@ This project contains:
    - [Jakarta Servlet](#jakarta-servlet)
    - [Jakarta Transactions](#jakarta-transactions)
+   - [Jakarta WebSocket](#jakarta-websocket)
 - [Building](#Building)
 - [Contributing](#Contributing)
+- [License](#license)
 - [Feedback](#Feedback)
 --- 
 ## Client IDEs
 
-Eclipse LSP4Jakarta is consumed by the following IDEs:
+Eclipse LSP4Jakarta is consumed by the following client IDEs.
 -  Eclipse IDE with [Liberty Tools for Eclipse](https://github.com/OpenLiberty/liberty-tools-eclipse#welcome-to-the-liberty-tools-for-eclipse-project)
+-  Visual Studio Code with [Liberty Tools for Visual Studio Code](https://github.com/OpenLiberty/liberty-tools-vscode#liberty-tools-for-visual-studio-code)
+-  IntelliJ IDEA with [Liberty Tools for IntelliJ IDEA](https://github.com/OpenLiberty/liberty-tools-intellij#liberty-tools-for-intellij-idea)
+
+## Features
+
+The langauge features currently offered by Eclipse LSP4Jakarta to Jakarata EE APIs are code snippets, diagnostics, and quick-fixes.
+
+Code snippets help you quickly generate common blocks of code.
+
+![Eclipse LSP4Jakarta code snippet to generate a new RESTful web services GET method](./docs/images/restful-ws-snippet.png)
 
-## Jakarta EE API Language Features
+Diagnostics indicate incorrect API usage and suggest associated quick-fixes to correct API usage.
 
-The following is a list of Eclipse LSP4Jakarta snippets offered for each [Jakarta EE API](https://jakarta.ee/specifications/). For a full list of language features (snippets, diagnostics and quick-fixes) currently offered by Eclipse LSP4Jakarta refer to the checked off items in [issue #16](https://github.com/eclipse/lsp4jakarta/issues/16). Language features will only be delivered for a given Jakarta EE API if that API is found on the project's classpath. Eclipse LSP4Jakarta is currently targeting Jakarta EE 9/9.1 (jakarta.x namespace). 
+![Eclipse LSP4Jakarta diagnostic and quick-fix for invalid RESTful web services usage](./docs/images/restful-ws-diagnostic-quick-fix.png)
+
+The following is a list of Eclipse LSP4Jakarta language features offered for each [Jakarta EE API](https://jakarta.ee/specifications/). Language features will only be delivered for a given Jakarta EE API if that API is found on the project's classpath. Eclipse LSP4Jakarta is currently targeting Jakarta EE 9/9.1 (jakarta.x namespace). 
  
 If you would like to see language feature assistance for a Jakarta EE API that is not yet listed here, please open an issue on the [issue tracker](https://github.com/eclipse/lsp4jakarta/issues).
 
+
+### Jakarta Annotations
+
+Eclipse LSP4Jakarta provides the following [Jakarta Annotations diagnostics](./jakarta.jdt/org.eclipse.lsp4jakarta.jdt.core/src/main/java/org/eclipse/lsp4jakarta/jdt/core/annotations/).
+
+| Diagnostic description | Severity | Associated quick-fix(s) |
+| --------- | ----- | ------------ |
+| The annotation @Generated must define the attribute 'date' following the ISO 8601 standard. | Error | N/A |
```

Comment:
sorry missed this the first time- for consistency, the statements in this column should statement the annotation name before the identifier "annotation", same with attributes eg 
```suggestion
| The @Generated annotation must define the 'date' attribute following the ISO 8601 standard. | Error | N/A |
```